src/parser.h

```diff
@@ -334,6 +333,9 @@ class parser_t : public std::enable_shared_from_this<parser_t> {
                     const job_group_ref_t &job_group = {},
                     block_type_t block_type = block_type_t::top);
 
+    /// An ffi overload of `eval(const wcstring &cmd, ...)` but without the extra parameters.
+    eval_res_t eval_string_ffi1(const wcstring &cmd);
```

Comment:
I think this one doesn't need the `1` suffix.
The other overload can be called `eval_parsed_source` or similar.

Reply:
I was thinking more for the future when we add another overload for this one that has the job_group parameter.